Libplanet.Explorer/Indexing/BlockChainIndexBase.cs

```diff
@@ -298,8 +298,16 @@ CancellationToken stoppingToken
             }
 
             var blockDigest = store.GetBlockDigest(indexEnumerator.Current)!.Value;
-            var txs = blockDigest.TxIds.Select(
-                txId => (ITransaction)store.GetTransaction(new TxId(txId))).ToArray();
+            var txs = blockDigest.TxIds.Select(txId =>
+            {
+                if (store.GetTransaction(new TxId(txId)) is { } tx)
+                {
+                    return tx;
+                }
+
+                var message = $"Cannot found transaction: '{txId}' in store.";
+                throw new InvalidOperationException(message);
+            }).ToArray();
```

Comment:
Seems unnecessarily verbose.

```suggestion
            ITransaction[] txs = blockDigest.TxIds
                .Select(txId => store.GetTransaction(new TxId(txId))
                    ?? throw new InvalidOperationException(
                        $"Could not find transaction with txid {txId} in store."))
                .ToArray();
```